tetrateci/create_istio_release.sh

```diff
@@ -109,7 +109,9 @@ if [[ -z ${TEST:-} ]]; then
     # if FIPS, need to use native go as boringgo as of now can't build archives for different platforms
     if [[ ${TAG} =~ "fips" ]]; then
         sudo rm -rf /usr/local/go
-        source ${BASEDIR}/tetrateci/setup_go.sh
+        source ${BASEDIR}/tetrateci/setup_boring_go.sh
+        # for fips build, only linux-amd64 artifacts are needed since linux-amd64 is the only fips compliant OS Arch in the list" 
+        sed -i 's/"linux-amd64"\s,\s"linux-armv7",\s"linux-arm64",\s"osx",\s"osx-arm64",\s"win"/"linux-amd64"/g' pkg/build/archive.go
```

Comment:
Fine for now.

If we want I think very likely upstream is okay with adding a field, in `manifest` struct,  https://github.com/istio/release-builder/blob/47173658f669cca3ede0491318fa0caa9c01ef1b/pkg/model/model.go#L176
to filter the arch/os to build for. And make the default behavior the same.

just a thought.